mne/viz/topomap.py

```diff
@@ -1168,7 +1168,9 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
     fig = axes[0].get_figure()
     fig.subplots_adjust(left=w_frame, right=1 - w_frame, bottom=0,
                         top=1 - top_frame)
-    time_idx = [np.where(evoked.times >= t)[0][0] for t in times]
+    # find first index that's >= (to rounding error) to each time point
+    time_idx = [np.where(_time_mask(evoked.times, t, None))[0][0]
```

Comment:
if this is a concern we should rewrite matplotlib and its backends in
cython or so ...

On Wed, Feb 17, 2016 at 11:16 AM, jaeilepp <notifications@github.com> wrote:

> In mne/viz/topomap.py
> <https://github.com/mne-tools/mne-python/pull/2895#discussion_r53144170>:
>
> > @@ -1168,7 +1168,9 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
> >      fig = axes[0].get_figure()
> >      fig.subplots_adjust(left=w_frame, right=1 - w_frame, bottom=0,
> >                          top=1 - top_frame)
> > -    time_idx = [np.where(evoked.times >= t)[0][0] for t in times]
> > +    # find first index that's >= (to rounding error) to each time point
> > +    time_idx = [np.where(_time_mask(evoked.times, t, None))[0][0]
>
> Few rounds of computations :)
> But this gets called max 20 times per plot, so I'm fine with either way.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2895/files#r53144170>.
>